deploy/helm/secret-operator/crds/crds.yaml

```diff
@@ -395,6 +395,18 @@ spec:
                 secretClassName:
                   description: The name of the SecretClass that the request concerns.
                   type: string
+                targetKind:
+                  default: ConfigMap
```

Comment:
I missed the boat on this, but I think `Secret` should be the default.
I assume ConfigMap was chosen as a default for good reason, so please continue.

Reply:
Oh I guess because that was the existing behaviour.

Reply:
Correct, existing behavior is a ConfigMap. If we would default to Secret it's breaking ;)
Also I recall someone in a discussion saying that we should try to be transparent that this is *not* confidential data.